### ISSUE TITLE
Review relation between abnormal susceptibility to glomerulonephritis and glomerulonephritis

### ISSUE THREAD

**Issue by anna-anagnostop** [mgijax/mammalian-phenotype-ontology#3818]
Currently, the MP has the following glomerulonephritis terms:

- glomerulonephritis [MP:0002743](https://www.informatics.jax.org/vocab/mp_ontology/MP:0002743) (under [cortical renal glomerulopathies](https://www.informatics.jax.org/vocab/mp_ontology/MP:0000523) and [kidney inflammation](https://www.informatics.jax.org/vocab/mp_ontology/MP:0001859)) with 235 annotations
- abnormal susceptibility to glomerulonephritis [MP:0021199](https://www.informatics.jax.org/vocab/mp_ontology/MP:0021199)  (under [abnormal renal glomerulus morphology](https://www.informatics.jax.org/vocab/mp_ontology/MP:0005325)) with deceased susceptibility MP:0021200 and increased susceptibility MP:0021201 child terms but zero annotations

Need to review our annotations to glomerulonephritis [MP:0002743](https://www.informatics.jax.org/vocab/mp_ontology/MP:0002743) to see if we need to:

- [ ] make increased susceptibility to glomerulonephritis a direct child of glomerulonephritis
- [ ] move any of our existing annotations to a more appropriate (e.g., decreased) susceptibility term
- [ ] tweak term label/add a comment to susceptibility terms to indicate whether induction of glomerulonephritis occurs by external agent/experimental means

**Comment by sbello** [comment]
Discussion with Leigh from HPO as HPO has terms for:

- Glomerulopathy (HP:0100820) defined as ‘Inflammatory or noninflammatory diseases affecting the glomeruli of the nephron.’ 
- Lobular glomerulopathy (HP:0008636), 
- Glomerulonephritis (HP:0000099)
- Glomerular sclerosis (HP:0000096)
all of these terms are siblings
So are glomerulonephritis and glomerulosclerosis actually 'diseases' affecting the glomeruli or not?

There is also the issue of having a term (cortical renal glomerulopathies) in the MP that is explicitly defined as a disease. This is an old term, possibly we should obsolete it. There are ~34 annotations directly to this term we could possibly review and move these notes but a quick glance does show several that mention disease or lesion or ..opathy in the note.
